src/customizations/volto/components/theme/View/LinkView.jsx

```diff
@@ -13,20 +13,20 @@ import config from '@plone/volto/registry';
 
 const LinkView = ({ token, content }) => {
   const history = useHistory();
-  const userIsSpidUser = useSelector(
-    (state) => state.users.user.roles.length === 0,
+  const isReader = useSelector((state) =>
+    state.users.user ? state.users.user.roles.length === 0 : true,
```

Comment:
```suggestion
    state.users?.user ? state.users.user?.roles?.length === 0 : true,
```
troppo?

Reply:
Ho fatto diverse prove e non sono mai stato nel caso in cui non c'era users, però direi che male non fa